src/Orchard.Web/Modules/Orchard.Lists/Views/Breadcrumbs.ContentItem.cshtml

```diff
@@ -4,10 +4,10 @@
     var breadcrumbs = new List<dynamic>();
     var containerAccessor = (Func<IContent, IContent>) Model.ContainerAccessor;
     do {
-        var displayText = content.Id > 0 ? Html.ItemDisplayText(content).ToString() : T("New {0}", content.ContentItem.TypeDefinition.DisplayName).ToString();
+        var displayText = content.Id > 0 ? (Html.ItemDisplayText(content) != null) ? Html.ItemDisplayText(content).ToString() : T("{0}", content.ContentItem.TypeDefinition.DisplayName).ToString() : T("New {0}", content.ContentItem.TypeDefinition.DisplayName).ToString();
```

Comment:
Replace `T("{0}", content.ContentItem.TypeDefinition.DisplayName)` by `content.ContentItem.TypeDefinition.DisplayName`. 

No translation is needed for a raw string.